ddtrace/internal/datadog/profiling/stack_v2/src/stack_v2.cpp

```diff
@@ -0,0 +1,71 @@
+#include "cast_to_pyfunc.hpp"
+#include "python_headers.hpp"
+#include "sampler.hpp"
+
+using namespace Datadog;
+
+static PyObject*
+_stack_v2_start(PyObject* self, PyObject* args, PyObject* kwargs)
+{
+    (void)self;
+    static const char* const_kwlist[] = { "min_interval", NULL };
+    static char** kwlist = const_cast<char**>(const_kwlist);
+    double min_interval_s = g_default_sampling_period_s;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|d", kwlist, &min_interval_s)) {
+        return NULL; // If an error occurs during argument parsing
+    }
+
+    Sampler::get().set_interval(min_interval_s);
+
+    return PyLong_FromLong(1);
```

Comment:
What feedback would we lose if we returned `None` instead?

Reply:
Ah, good point.  Originally this returned a bool, but right now there's nothing to do about it.  Agree.